src/plugins/auto_batch/CMakeLists.txt

```diff
@@ -30,5 +30,5 @@ if(ENABLE_FUNCTIONAL_TESTS)
 endif()
 
 if(ENABLE_TESTS)
-    add_subdirectory(tests/unit)
+#    add_subdirectory(tests/unit)
```

Comment:
Why?

Reply:
the unite test for auto batch will be prepared in another PR

Reply:
Let's avoid regressions...